docs/source/developers/extensions.rst

```diff
@@ -156,6 +156,10 @@ The basic structure of an ExtensionApp is shown below:
             ...
             # Change the jinja templating environment
 
+        def stop_extension(self):
```

Comment:
Per general comment, it would be great if we could make this async so we could gather these "simultaneously".

Unrelated, but could you please fix the typo in [`ExtensionsApp` above](https://github.com/jupyter-server/jupyter_server/pull/526/files#diff-1568ad787ad260c37ff89b0f13fc867e152909f1461b64af312531e4f3a2c49bR106) while we're in here?  Thank you.

Reply:
Changed `s/initialize_setting/initialize_settings/`